tests/schema/tests.py

```diff
@@ -2676,6 +2676,39 @@ class Meta:
         with self.assertRaises(IntegrityError):
             Tag.objects.create(title="bar", slug="foo")
 
+    def test_unique_with_fk_and_condition_multiple_times(self):
+        """
+        Tests adding and removing a unique constraint to ForeignKey
+        with a condition multiple times.
+        """
```

Comment:
This docstring is unnecessary when we switch to checking the number of constraints.